florist/api/index.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/api/python")
+def hello_world():
```

Comment:
Not a big deal, but I am curious how this passed mypy pre-commit checks when the return type is not specified? Do you have any idea?

Reply:
Good question. Will investiagate.

Reply:
I guess if you don't add annotation it infers the return type as Any and it passes based on the [mypy documentation](https://mypy.readthedocs.io/en/stable/common_issues.html#no-errors-reported-for-obviously-wrong-code). Not that this code is incorrect, but I think this may be relevant.

Reply:
That's not super great, the function is clearly returning a string but mypy doesn't pick that up by default. I've copied the config from FL4Health and now it complains about it :)

Reply:
Hooray for the FL4Health configuration being sort of correct! 😂